src/test/java/de/tum/in/www1/artemis/exam/ExamUtilService.java

```diff
@@ -428,6 +431,21 @@ public StudentExam addStudentExamWithUser(Exam exam, User user) {
         return studentExam;
     }
 
```

Comment:
Documentation for other developers would be helpful.

Otherwise, we always will have to check if this has any other side-effects etc.